### PR TITLE
Bump Rails, Rack and jQuery Rails for security fixes

### PR DESCRIPTION
**Pull request by boffbowsh**
Vulnerabilities addressed:

- CVE-2015-3225
https://groups.google.com/forum/#!topic/rubyonrails-security/gcUbICUmKMc
- CVE-2015-1840
https://groups.google.com/forum/#!topic/rubyonrails-security/XIZPbobuwaY
- CVE-2015-3227
https://groups.google.com/forum/#!topic/rubyonrails-security/bahr2JLnxvk

Also adds a workaround for - CVE-2015-3226
https://groups.google.com/forum/#!topic/rubyonrails-security/7VlB_pck3hU

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ end
 
 gem 'erubis'
 gem 'govuk_admin_template', '2.1.0'
+gem 'jquery-rails', '~> 3.1.3'
```
Comment (alext) [inline Gemfile]: Instead of adding this, you could bump govuk_admin_template to `~> 2.3.1`

Comment (boffbowsh) [inline Gemfile]: Was reluctant to bump it a couple minor versions. Is it safe?

Comment (alext) [inline Gemfile]: We follow semver, so it should be.

**Comment by boffbowsh** [comment]
Updated.

**Pull request merged**